src/analysis/retail/monk/mistweaver/modules/spells/InvokeChiJi.tsx

```diff
@@ -426,7 +436,13 @@ class InvokeChiJi extends BaseCelestialAnalyzer {
                 <SpellLink id={SPELLS.GUST_OF_MISTS_CHIJI.id} />.
               </li>
               <li>
-                {formatNumber(this.envelopHealing)} healing from{' '}
+                {formatNumber(this.envelopHealing)}{' '}
+                <SpellLink id={TALENTS_MONK.ENVELOPING_BREATH_TALENT.id} /> healing from{' '}
+                <SpellLink id={TALENTS_MONK.ENVELOPING_BREATH_TALENT.id} />.
```

Comment:
Since the new talent is called Celestial Harmony, we might want to change the talent object accordingly and this will make this tooltip make a bit more sense

Reply:
Will make a follow up PR for this after the script is run to update TALENTS